packages/react/tests/e2e-storybook/cypress/integration/Footer/Footer.e2e.js

```diff
@@ -114,82 +122,123 @@ describe('Footer | default (desktop)', () => {
   });
 });
 
-describe('Footer | default (mobile)', () => {
+describe('Footer | Short (desktop)', () => {
   beforeEach(() => {
-    cy.visit(`/${_pathDefault}`);
-    cy.viewport(320, 780);
+    cy.visit(`${_pathShort}`);
+    cy.viewport(1280, 780);
   });
 
-  it('should have interactable url for IBM logo', () => {
-    cy.get('[data-autoid="dds--footer-logo__link"]').then($link => {
+  it('should display clickable IBM logo', () => {
+    cy.get('a.bx--footer-logo__link').then($link => {
       const url = $link.prop('href');
       expect(url).not.to.be.empty;
     });
-  });
-
-  it('should load locale modal', () => {
-    const localeButton = cy.get(`[data-autoid="dds--locale-btn"]`);
-    localeButton.click();
 
     cy.screenshot();
+
+    cy.percySnapshot('Footer | Short | load clickable IBM logo', {
+      widths: [1280],
+    });
   });
 
-  it('should load the Americas region with its languages and locations', () => {
-    const localeButton = cy.get(`[data-autoid="dds--locale-btn"]`);
-    localeButton.click();
+  it('should open locale modal with 4 geos when clicked on locale button', () => {
+    cy.get('button[data-autoid="dds--locale-btn"]').click();
 
-    cy.get('[data-region="am"]').click();
+    cy.wait(500);
 
-    cy.get('.bx--locale-modal__locales').should('have.length', 35);
+    cy.get('div[data-autoid="dds--locale-modal"]').should('have.attr', 'open');
+
+    cy.get('.bx--locale-modal [data-region]').should('have.length', 4);
 
     cy.screenshot();
 
-    // Take a snapshot for visual diffing
-    cy.percySnapshot('Footer | Americas region selected', {
-      widths: [1280],
-    });
+    cy.percySnapshot(
+      'Footer | Short | open locale modal with 4 geos when clicked on locale button',
+      {
+        widths: [1280],
+      }
+    );
   });
 
-  it('should be able to search with keywords for locations and languages', () => {
-    const localeButton = cy.get(`[data-autoid="dds--locale-btn"]`);
-    localeButton.click();
+  it('should display the specific locations and languages of a selected geo', () => {
+    cy.get('button[data-autoid="dds--locale-btn"]').click();
 
-    cy.get('[data-region="am"]').click();
-    cy.get('[data-autoid="dds--locale-modal__filter"]').type('mexico', {
-      force: true,
-    });
+    cy.wait(500);
 
-    cy.get(
-      '.bx--locale-modal__locales:not(.bx--locale-modal__locales-hidden) > div'
-    )
-      .first()
-      .then(e => {
-        expect(e.text()).to.equal('Mexico');
+    cy.get('.bx--locale-modal [data-region="mea"]')
```

Comment:
I think the only thing I'd ask is if there are stable selectors ([data-autoid="*"]) that are available and can be used instead of classes? Those would probably be more reliable long-term.

Reply:
Hey Jeff, I removed some of the class selectors. But I leave some especially in the search results since they don't have stable selectors. Let me know if those are to be removed as well.